adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java

```diff
@@ -596,7 +597,7 @@ public void testGetAuthTokenInBackgroundVerifyErrorMessageBadAuth()
         String acctName = "testAcct123";
         Object authRequest = createAuthenticationRequest("https://login.windows.net/omercantest", "resource", "client",
                 "redirect", acctName.toLowerCase(Locale.US), PromptBehavior.Auto, "", UUID.randomUUID(), false);
-        setMockProxyForErrorCheck(brokerProxy, acctName, AccountManager.ERROR_CODE_BAD_AUTHENTICATION,
+        setMockProxyForErrorCheck(brokerProxy, acctName, AccountManager.ERROR_CODE_BAD_ARGUMENTS,
```

Comment:
ERROR_CODE_BAD_AUTHENTICATION is API level 18. if we say we support 14 this might crash or cause unexpected behavior. Does this not look correct?